business/checkers/services/port_mapping_checker.go

```diff
@@ -40,6 +41,10 @@ func (p PortMappingChecker) Check() ([]*models.IstioCheck, bool) {
 		}
 	}
 
+	// Ignoring istio-system Services as some ports are used for debug purposes and not exposed in deployments
+	if config.IsIstioNamespace(p.Service.Namespace) {
+		return validations, len(validations) == 0
```

Comment:
Minor comment.
I'd put a "log.debug" ("info" is too much) entry here, that would help to not miss this edge on the logic.

Reply:
Is "Skipping Port matching check for Service %s from Istio Namespace %s" enough for message?

Reply:
Sure, just a debug line to not miss that the validations are entering in that edge :-).
Sorry if my previous comment was not good.

Reply:
got it, for nothing, thank you for checking